src/doc/src/reference/config.md

```diff
@@ -664,6 +667,41 @@ Controls how often we display a notification to the terminal when a future incom
 * `always` (default): Always display a notification when a command (e.g. `cargo build`) produces a future incompat report
 * `never`: Never display a notification
 
+### `[cache]`
+
+The `[cache]` table defines settings for cargo's caches.
+
+#### Global caches
+
+When running `cargo` commands, Cargo will automatically track which files you are using within the global cache.
+Periodically, Cargo will delete files that have not been used for some period of time.
+It will delete files that have to be downloaded from the network if they have not been used in 3 months. Files that can be generated without network access will be deleted if they have not been used in 1 month.
```

Comment:
nit: Forgot to add a line break maybe?

And ditto in doc changes in this PR.

(Assuming we follow https://sembr.org)

```suggestion
It will delete files that have to be downloaded from the network if they have not been used in 3 months.
Files that can be generated without network access will be deleted if they have not been used in 1 month.
```